### PR TITLE
Added Buttons and State in the Embedded Form

### PR DESCRIPTION
**Pull request by rohanr2001**

### CODE REVIEW

File: .vscode/launch.json
Comment (rohanr2001) [inline .vscode/launch.json:1]: The skipFiles property is used in debugging configurations to skip dependencies.
This avoids the debugger going in the "node_modules" folder.

File: packages/shared/utils/fileUpload.ts
Comment (rohanr2001) [inline packages/shared/utils/fileUpload.ts:1]: Formated by the code editor, though no changes in the flow or functionality of the code.

File: packages/web/src/components/embed/EmbedFormTab.tsx [hunk 1]
```diff
@@ -21,8 +21,6 @@ import FormFields from '../form2/FormFields';
 
 // EMBED SPECIFIC LIBS
 import { CopytoClipboard, getEmbedLink } from './embedLibs';
```
Comment (rohanr2001) [inline packages/web/src/components/embed/EmbedFormTab.tsx:23]: Unnecessary Imports

**Comment by appdatasoft** [comment]
https://github.com/Codemarket2/boossti-frontend/assets/41810092/322d547a-5039-4908-9c6b-552d56b1e1f1

**Pull request merged**